packages/salesforcedx-utils/src/cli/commandOutput.ts

```diff
@@ -23,9 +28,12 @@ export class CommandOutput {
       (resolve: (result: string) => void, reject: (reason: string) => void) => {
         execution.processExitSubject.subscribe(data => {
           if (data !== undefined && String(data) === '0') {
-            return resolve(this.stdoutBuffer);
+            return resolve(stripAnsiInJson(this.stdoutBuffer, hasJsonEnabled));
```

Comment:
Two different versions of the same code, so the this is `deja vue` all over again is appropriate.